ZoolandiaRazor/Views/Habitat/Details.cshtml

```diff
@@ -0,0 +1,16 @@
+﻿
+@{
+    ViewBag.Title = "Habitat Details";
+}
+
+<h2>Habitat Details</h2>
+
+<h3>@ViewBag.habitat.Name</h3>
+
+<h4>Our Residents include: </h4>
+
+@foreach(var hab in ViewBag.habitat)
+{
+    <p></p>
```

Comment:
No wait it's the details page. Should loop through the residents. And print those out.